File: Itido-BMS/dart/bms_dart/lib/src/models/comment.dart
class Comment {
  int id;
  String title;
  String body;

  Comment({this.title, this.body});

  factory Comment.fromJson(json) {
    if (json == null) return null;
    return Comment._fromJson(json);
  }

  Comment._fromJson(json)
      : this.id = json['id'],
        this.title = json['title'],
        this.body = json['body'];

  Map<String, dynamic> toMap() => {
        'id': this.id,
        'title': this.title,
        'body': this.body,
      };
}

File: Itido-BMS/RentAPI/Rent/Models/Projects/Comment.cs
using System;
using System.Linq.Expressions;

namespace Rent.Models.Projects
{
    public class Comment
    {
        public int ID { get; set; }
        public int ProjectItemID { get; set; }
        public virtual ProjectItem ProjectItem { get; set; }
        public string Title { get; set; }
        public string Body { get; set; }

        static public Expression<Func<Comment, dynamic>> BasicDTO()
        {
            return v => v != null ?
            new
            {
                v.ID,
                v.Title,
                v.Body,
            } : null;
        }
    }
}
